assets/app/view/tools.rb

```diff
@@ -5,8 +5,11 @@
 module View
   class Tools < Snabberb::Component
     include GameManager
+    include Actionable
 
     needs :game, store: true
+    needs :user
+    needs :confirming_conclude_game, store: true, default: false
```

Comment:
can you rename this to something shorter

Reply:
how does this work, are you able to end games that are already ended?

Reply:
nope, it checks to see if the game is already finished before showing the option to end the game.